src/components/compound/DlSearches/DlSmartSearch/components/DlSmartSearchInput.vue

```diff
@@ -349,7 +349,7 @@ export default defineComponent({
             setInputValue(
                 clearPartlyTypedSuggestion(input.value.innerText, stringValue)
             )
-            setCaret(input.value)
+            setCaretAtTheEnd(input.value)
```

Comment:
@prodan-eric why set at end ? 
we need to also modify the code in the middle. would this throw the cursor to the end aswell ?

Reply:
@prodan-eric are we using the new selection util also in the smart search ?

Reply:
No, but I think it would be usable there

Reply:
@prodan-eric its ok, a task for another day i believe